ClientSideEncryptionToCPKN/ClientSideKeyVaultKeyToCustomerProvidedKey/samplesrc/Migration.cs

```diff
@@ -0,0 +1,132 @@
+            };
+
+            //Reupload Blob with Server Side Encryption using blob with Blob Client Options
+            blobClient = new BlobClient(
+                connectionString,
+                containerName,
+                fileName,
+                blobClientOptions);
+            using FileStream uploadFileStream = File.OpenRead(downloadFilePath);
+            blobClient.Upload(uploadFileStream, true);
+            uploadFileStream.Close();
+        }
+
+        /*
+         * Program uploads an example client side encrypted blob, and migrates it to server side encryption using Customer Provided Keys         
+         *
+         * NOTE: This program requires the following to be stored in the App.Config file:
+         * Azure Active Directory Tenant ID - tenantId
+         * Service Principal Application ID - clientId
+         * Service Principal Password - clientSecret
+         * Azure Subscription ID - subscriptionId
+         * Resource Group Name - resourceGroup
+         * Storage Account Name - storageAccount
+         * Storage Account Connection String- connectionString
+         * Key Vault Key Uri - keyVaultKeyUri
+         * Key Wrap Algorithm - keyWrapAlgorithm
+         * 
+         * NOTE: This program uses names from Constants.cs, which should be edited as needed
+         * 
+         */
+        static void Main()
+        {
+            //Credentials of Service Principal
+            TokenCredential credential =
+                new ClientSecretCredential(
+                    Constants.tenantId,
+                    Constants.clientId,
+                    Constants.clientSecret
+                    );
+
+            //Get bytes for customer provided key
+            byte[] localKeyBytes = ASCIIEncoding.UTF8.GetBytes(Constants.customerProvidedKey);
+
+            //File Path for local file used to download and reupload
+            string localPath = "./data" + Guid.NewGuid().ToString() + "/";
+            Directory.CreateDirectory(localPath);
+            string localFilePath = Path.Combine(localPath, Constants.fileName);
+
+            //Get Uri for Key Vault key
+            Uri keyVaultKeyUri = new Uri(Constants.keyVaultKeyUri);
+            //Create CryptographyClient using Key Vault Key
+            CryptographyClient cryptographyClient = new CryptographyClient(keyVaultKeyUri, credential);
+            //Set up Client Side Encryption Options used for Client Side Encryption
+            ClientSideEncryptionOptions clientSideOptions = new ClientSideEncryptionOptions(ClientSideEncryptionVersion.V1_0)
+            {
+                KeyEncryptionKey = cryptographyClient,
+                KeyWrapAlgorithm = Constants.keyWrapAlgorithm
+            };
+
+            //Create Blob Service Client
+            BlobServiceClient blobServiceClient = new BlobServiceClient(Constants.connectionString);
+
+            //Run Setup Function that creates and example container and blob
+            Setup.SetupForExample(
```

Comment:
talked to amanda offline and we agreed to keep this migration.cs  as a sample and provide another file so that user can directly run that file instead of reading through the code and find out they don't actually need to call SetupForExample or something else.